src/index.html

```diff
@@ -7,12 +7,104 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link references an SCSS file ('./styles/index.scss'). Browsers cannot directly process SCSS files. Ensure that this SCSS file is compiled into a CSS file, and link to the resulting CSS file instead.